app/add_note.py

```diff
@@ -160,6 +160,24 @@ def post(self):
             person.latest_status not in ['believed_alive', 'is_note_author']):
             UserActionLog.put_new('mark_alive', note, person.primary_full_name)
 
+        if note.status == 'is_note_author':
```

Comment:
1. Use for-loop to simplify this code. You can also get a list of statuses from this constant:
https://github.com/google/personfinder/blob/master/app/const.py#L259

    * If you find yourself repeating very similar code many times, it's a bad sign. See:
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

1. Consider putting this code in Note.put_new so that you can also remove another dup code in create.py.
1. Currently db.put() is called twice, one for "notes", once for "believed_alive" for a single note, right? Can you call db.put() just once for both?